src/allreduce_base.cc

```diff
@@ -13,6 +13,10 @@
 #include <cstring>
 #include "./allreduce_base.h"
 
+#ifdef _OPENMP
+#include <omp.h>
+#endif  // _OPENMP
+
```

Comment:
since open mp has removed, but a lot of headers like this are still here?

Reply:
fixed.